code/modules/species/species.dm

```diff
@@ -348,6 +354,10 @@ var/global/const/DEFAULT_SPECIES_HEALTH = 200
 
 	. = ..()
 
+	if(config.grant_basic_darksight)
+		darksight_range = max(darksight_range, 2)
```

Comment:
I think the basic darksight strengths (2 and 0.05, specifically) should also be configuration variables, with these as the defaults. (I also think it'd still be worthwhile to keep `grant_basic_darksight` around, since sometimes you just want to be able to toggle something off without clobbering the preferred settings for it.)